### ISSUE TITLE
Quarterly goals for Community Experience team

### ISSUE THREAD

**Issue by alecmccrindle** [CodeYourFuture/CYF-Staff-Tickets#302]

**Comment by alecmccrindle** [comment]
**Proposed**

1. Fundraising - raise £100k from individual donors 
2. Awareness - reach new audiences with CYF story
3. Volunteering - new volunteers and make the process sustainable  

Anything to add, remove or change @MarianaVazquezCYF? 

Any feedback or ideas @caromelendezl? Maybe a goal about driving corporate connections too? We can build that into the Awareness campaign. 

@lilianabermudes, what is your feedback?

**Comment by lilianabermudes** [comment]
Hi Alec,

For the volunteering one, I'm thinking if we should split per group?, considering **cohort founders** (--> we need more people in the WM and NW, for instance), Digital Literacy etc.

**Comment by MarianaVazquezCYF** [comment]
I would add: 

1. Volunteers: Global team and Proms are using the guide to recruit volunteers
2. Community Engagement: Strengthen the volunteers communities - Key result: one monthly team bonding activity per region
3. Increase word of mouth - Key result: 1 Volunteer per region introduce us to their companies 
4. Job fair - Key result - 6 companies attend our job fair!

**Comment by lilianabermudes** [comment]
Mariana, is point 4 London only?

**Comment by MarianaVazquezCYF** [comment]
yes for this quarter Lili !:D

**Comment by alecmccrindle** [comment]
Hi all, I believe these are the final goals. I'll update the Q4 goals deck.

- 2 or 3 x WM6 cohort founders
- 1 x new inbound company hiring contact per week from November onwards
- Start Sheffield chapter
- Community-building training in NW, WM and Scot
- Bring 8 tech leaders to CYF through events in Q4  
- First 20k in donations and host 2 x events
- 30% growth in website visits yoy
- 20% growth in Linkedin followers in Q4

**Comment by MarianaVazquezCYF** [comment]
@alecmccrindle and the job fair :D ! and we will not have fundraising events this quarter btw!

**Comment by alecmccrindle** [comment]
Hi @MarianaVazquezCYF - the two events are the Tech Confessions and Women in Tech brunch, if that's what you mean?

**Comment by MarianaVazquezCYF** [comment]
yes but they are not funraising events, they are part of the strategy to bring more leaders @alecmccrindle
